src/Xablu.WebApiClient/RestApiClient.cs

```diff
@@ -22,7 +22,7 @@ public class RestApiClient
 
         public RestApiClient(string apiBaseAddress)
```

Comment:
Is the `apiBaseAddress` parameter needed?

Reply:
Yes it is mendatory, although this constructor is more of an "convenience" constructor for developers that want to construct it themselves and not use the `RestApiClientOptions` object (this could be useful for developers that want to register the `RestApiClient` with their IoC container, instead of using the `CrossRestApiClient` class).